index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        
+        <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:300,400,400i|Nunito:300,300i" rel="stylesheet">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/png" href="img/favicon.png">
+        
+        <title></title>
+    </head>
+
+    <body>
+        <main class="Main-container">
+            <header class="Header">
+                <nav class="container">
+                    <div class="logoplaceholder">
+                        <a href="#" class="logo__link"><h1 class="logo">Welcome</h1></a>
```

Comment:
- [ ] As mentioned by a previous reviewer please don't use an `h1` tag in this section, `h1` tags are usually reserved for important topics on a webpage, please see this [>>link](https://www.reliablesoft.net/h1-tag/) for more details. You can instead use a `span` tag instead of `h1`